packages/clients/wh/university-submission-website/src/styles/global.css

```
@@ -0,0 +1,69 @@
+  color: white;
+  background: black;
+}
+.dark ::selection {
+  color: white;
+  background: black;
+}
+.select-none {
+  -webkit-tap-highlight-color: transparent;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)